### ISSUE TITLE
ArviZ fails to load when scipy.fft is imported

### ISSUE THREAD

**Issue by sethaxen** [arviz-devs/ArviZ.jl#188]
Our daily CI runs started spontaneously failing 14 days ago, i.e. going from https://github.com/arviz-devs/ArviZ.jl/runs/6517258108, which passes to https://github.com/arviz-devs/ArviZ.jl/runs/6533529487, which fails. It fails upon precompilation of the package and emits the error message:

```
The Python package arviz could not be imported by pyimport. Usually this means
that you did not install arviz in the Python version being used by PyCall.
```

No commits were made to ArviZ between these runs. Both runs use the same operating system version, same Julia and Python versions, and same PyCall and Conda versions. There were no new releases to Python ArviZ in that time period.

**Comment by sethaxen** [comment]
Found it! Seems to be an issue with loading the latest release of scipy in Julia's conda installation. See https://github.com/JuliaPy/PyCall.jl/issues/990

We could temporarily work around this by manually installing scipy v1.8.0 in our conda environment.

**Comment by sethaxen** [comment]
Reopening so users can find this issue.

There are 2 workarounds. First, you can launch Julia like this
```bash
$ LD_PRELOAD=$HOME/.julia/conda/3/lib/libstdc++.so julia
```
To avoid having to do this every time, you can add the following to your `.bashrc` file:
```bash
export LD_PRELOAD=$HOME/.julia/conda/3/lib/libstdc++.so
```

Second, since the issue is with conda's scipy version, you can pip install scipy instead:
```bash
$ ~/.julia/conda/3/bin/python -m pip install --force-reinstall --no-deps scipy
```

**Comment by sethaxen** [comment]
Fixed by #290
